src/components/shared/StackedData.js

```diff
@@ -18,12 +18,14 @@ const DivWrapper = styled.div`
 `;
 
 const Value = styled.span`
+  color: ${props => props.theme.global.textColor};
```

Comment:
I'd convert these to the theming style we use in ucl, and actually just going to throw out 2 things to consider before that. 
1. Does this not adopt the fontColor from Global styles?
2. If not, is that something we need to fix, or would we rather have it be explicit here?
2a. If explicit, because textColor is a very common theme syntax, this feels like it should live in the shared in ucl.

Reply:
We should take some time to look at this over a zoom. The StackedData component wasn't getting global styles like everything else, which is why I hacked this in here. I would prefer we fix this, rather than add the lines of code I did above.